src/app/@ansyn/map-facade/components/map-search-box/map-search-box.component.less

```diff
@@ -5,7 +5,7 @@
 
 :host {
 	position: absolute;
-	top: @compass-top + @compass-width / 2 + 3; // middle of compass
+	top: calc(@compass-top + @compass-width / 2 + 3px); // middle of compass
```

Comment:
Why this change?

(I see that the "left" definition below is similar, and you didn't change it)

Reply:
`calc` no longer accepts arguments without unit (so it needs 3px instead of 3)

the "left" definition below doesn't use the `calc` function therefore it's capable of dealing without units

Reply:
But there was no `calc` before. You added the `calc`. Why did you add it?

Reply:
fixed